crates/bevy_gizmos/src/gizmos.rs

```diff
@@ -24,6 +24,9 @@ pub(crate) struct GizmoStorage {
 }
 
 /// A [`SystemParam`](bevy_ecs::system::SystemParam) for drawing gizmos.
+///
+/// They are drawned in immediate mode, which means they will be rendered only for
```

Comment:
```suggestion
/// They are drawn in immediate mode, which means they will be rendered only for
```

Reply:
Damn it XD

It was 2am when I wrote this I was like "hmm that doesn't sound right", I searched and DDG was like "yeah okay sure"
![image](https://github.com/bevyengine/bevy/assets/134181069/a5dc1c72-c89b-4897-877d-556fcfbf9a69)